index.html

```diff
@@ -2019,8 +2057,8 @@ <h2>
         </h2>
 
         <p>
-When producing and consuming DID documents that are in plain JSON (as noted by
-the document metadata), the following rules MUST be followed.
+When producing and consuming DID documents that are in plain JSON (as indicated by
+a <code>content-type</code> of <code>application/json+did</code> in the resolver metadata), the following rules MUST be followed.
```

Comment:
<code>application/json+did</code> does not sound right. Shouldn't it be <code>application/did+json</code>?

Reply:
That's part of the discussion under #255

Reply:
Yes, this is wrong per RFC6838 and should be fixed. Thanks for catching that @iherman.

The specific section is here:

https://tools.ietf.org/html/rfc6838#section-4.2.8

More elaboration here:

https://tools.ietf.org/html/rfc6839

Reply:
Fixed in 4f1c79ce416dc2564e693d76cd63ede0242ccd80.